help_to_heat/templates/frontdoor/schemes.html

```diff
@@ -2,52 +2,186 @@
+    <p
+      class="govuk-body">{{ _("Based on your answers, the Great British Insulation Scheme might be suitable for you.") }}</p>
+    <div class="govuk-warning-text">
+      <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+      <strong class="govuk-warning-text__text">
+        <span class="govuk-visually-hidden">Warning</span>
+        {{ _("Please read the following details carefully") }}
+      </strong>
+    </div>
+    <p
+      class="govuk-body">{{ _("The Great British Insulation Scheme is delivered by energy suppliers and their installation partners. They will assess your home, either by a desktop assessment or by arranging a visit to your home. They will determine if they can offer you insulation. ") }}</p>
+    <h2 class="govuk-heading-m">{{ _("What you could be eligible for") }}</h2>
+    {% if text_flags["park_home_text"] %}
+      <p class="govuk-body">{{ _("Based on your answers, you may be able to get the following:") }}</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>{{ _("park home insulation") }}</li>
+      </ul>
+      <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title"
+         data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header"></div>
+        <p class="govuk-notification-banner__heading"></p>
+        <div class="govuk-notification-banner__content">
+          <p class="govuk-heading-m">{{ _("Your contribution") }}</p>
+          <p class="govuk-body">
+            <p class="govuk-body">
+              {{ _("Park home insulation may not be available from %(supplier)s or their installation partner. If it is available, you would likely have to pay towards the cost of installing it. You might be asked to contribute between £1,000 - £10,000.") % { "supplier": supplier_name }}}
+            </p>
+            </div>
+    </div>
+    {% else %}
+      <p class="govuk-body">{{ _("Based on your answers, you may be able to get one of the following:") }}</p>
+      <ul class="govuk-list govuk-list--bullet">
+        {% if text_flags["cavity_wall_text"] %}
+          <li>{{ _("cavity wall insulation") }}</li>
+        {% endif %}
+        {% if text_flags["loft_insulation_text"] %}
+          <li>{{ _("loft insulation") }}</li>
+        {% endif %}
+      </ul>
+      {% if text_flags["loft_insulation_text"] or  text_flags["cavity_wall_text"] %}
+        <p class="govuk-body">{{ _("The following measures are less likely to be viable, but depending on the assessment of your home, you may otherwise be able to get one of:") }}</p>
+      {% endif %}
+      <ul class="govuk-list govuk-list--bullet">
+        {% if text_flags["solid_wall_text"] %}
+          <li>{{ _("solid wall insulation (internal or external)") }}</li>
+        {% endif %}
+          <li>{{ _("flat or pitched roof insulation") }}</li>
+          <li>{{ _("solid or under floor insulation") }}</li>
```

Comment:
these are over padded
```suggestion
        <li>{{ _("flat or pitched roof insulation") }}</li>
        <li>{{ _("solid or under floor insulation") }}</li>
```